.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,54 @@
+name: Bug report 🐞
+description: File a bug report
+title: "🐞[Bug]: "
```

Comment:
For title, we can use the pattern used in commit messages. You can take a look at `changelog.config.js` file for more information.

Reply:
@growupanand, Do you want me to change title from this `🐞[Bug]: ` to this `🐛fix: `  like that or what? Please explain it clearly.

Reply:
Yes, as you can see in commit lists.

![image](https://github.com/growupanand/ConvoForm/assets/29487686/37f12231-9571-4b22-9e25-0f978e3e7783)